examples/base/mod.js

```diff
@@ -1,3 +1,10 @@
+const obj = {
+  'bar': 'foo',
+};
+
 export function foo() {
-  return 'foo';
+  // This is a legacy style from a time when Closure Compiler would mangle property names.
+  // TODO: Confirm this is still needed and remove the `quoteProps: 'preserve'`
+  // option for prettier if not.
+  return obj['bar'];
```

Comment:
I think we have determined that quoting object properties is no longer necessary.  I don't think we need to change our convention now, but if we want to in the future, we can remove the `quoteProps: 'preserve'` configuration.